### ISSUE TITLE
Percent-encoded reference property IRIs getting percent-encoded again in href attribute

### ISSUE THREAD

**Issue by kubeluk** [kianschmalenbach/rdf-browser#41]
Hi!

I'm using the extension to browse a [Direct Graph](https://www.w3.org/TR/rdb-direct-mapping/#definition) and realized that reference property IRIs are not working properly. This is an example:
<img width="1510" alt="image" src="https://github.com/user-attachments/assets/cb494c03-df54-433c-8647-6249a5fd8401">

I've found out that this is because percent-encoded IRIs are getting percent-encoded again, when the `href` attribute is being generated. See the generated HTML for the reference property IRI:
```
<span class="object" indent="114">
    <span class="uri">
        <a href="http://localhost:3000/rdf-graph-store/sport/id%253D100">&lt;http://localhost:3000/rdf-graph-store/sport/id%3D100&gt;</a>
    </span>
</span>
```

I assume this should not be the expected behavior? If I understand the code correctly, the issue results from [here](https://github.com/kianschmalenbach/rdf-browser/blob/525853d9f285a6b24f6d76483de75e23e6ad7969/src/bdo/resource.js#L87). Maybe it is possible to skip this line, if an already percent-encoded IRI is detected? Thanks for your help!

Best,
Lukas

**Comment by aharth** [comment]
Yes, I see a similar behaviour: already urlencoded arguments in query parts of a URI get encoded again.
